vagrant/provisioning/roles/activemq/tasks/main.yml

```diff
@@ -25,6 +25,22 @@
     - "{{ root_folder }}/install/activemq"
     - "{{ root_folder }}/app/activemq/conf"
 
+#- name: Set Java alternatives to OpenJDK 17
+#  command: alternatives --set java /usr/lib/jvm/java-17-amazon-corretto/bin/java
+#  become: yes
+
+- name: Register Java 17 as an alternative
+  become: yes
+  ansible.builtin.command:
```

Comment:
Couple of things.
1. You don't need ansible.builtin, that is the full package, and only command will be sufficient. 
2. I've tried to run it (Rocky8 VM), but was not able to match the desired outcome:
![image](https://github.com/ArkCase/arkcase-ce/assets/48589462/ed56873f-190e-4b81-b9d4-ee9e834fbdac)